soliket/tests/test_mflike.py

```diff
@@ -95,17 +97,35 @@ def test_mflike(self):
         # w/out cobaya
 
         camb_cosmo = cosmo_params.copy()
-        camb_cosmo.update({"lmax": 9000, "lens_potential_accuracy": 1})
+        lmax = 9000
+        camb_cosmo.update({"lmax": lmax, "lens_potential_accuracy": 1})
         pars = camb.set_params(**camb_cosmo)
         results = camb.get_results(pars)
         powers = results.get_cmb_power_spectra(pars, CMB_unit="muK")
         cl_dict = {k: powers["total"][:, v] for
                    k, v in {"tt": 0, "ee": 1, "te": 3}.items()}
+
         if not self.orig:
+
+            BP = soliket.BandPass()
+            FG = soliket.Foreground()
             TF = soliket.TheoryForge_MFLike()
-            #Dls = cl_dict
-            #params = cosmo_params + nuisance_params
-            dlobs_dict = TF.get_modified_theory(cl_dict, **nuisance_params)
+
```

Comment:
Is there a way to write `MFLike` so that it just directly uses the `BandPass` and `Foreground` likelihoods, rather than requiring the `TheoryForge` middleman?  Probably more thoughts later...

Reply:
Ideally, the TF module is the one using BandPass and Foreground to build theoretical predictions. Likelihood blocks should only call TF with their specific requirements. Agrees, more thoughts needed